utils/compare_minions.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+import salt.config
+import salt.loader
+
+import json
+import json_delta
+
+try:
+    from colorama import Fore, Back, Style, init
+    init()
+except ImportError:  # fallback so that the imported classes always exist
+    class ColorFallback():
+        __getattr__ = lambda self, name: ''
+    Fore = Back = Style = ColorFallback()
+
+def color_diff(diff):
+    '''Colorize lines in the diff generator object'''
+
+    for line in diff:
+        if line.startswith('+'):
+            yield Fore.GREEN + line + Fore.RESET
+        elif line.startswith('-'):
+            yield Fore.RED + line + Fore.RESET
+        else:
+            yield line
+
+class Minion():
+
+    default_config = '/opt/{name}/minion'
+
+    def __init__(self, name=None, config=None):
+        if not config:
+            config = self.default_config.format(name=name)
+        self.config=config
+
+        __opts__ = salt.config.minion_config(self.config)
```

Comment:
these `__variables__` look like python magic methods and fields. But they are not. So i'd replace them with normal var defs to avoid confusing